README.md

```diff
@@ -1,27 +1,29 @@
 # Minehut Meta
 Welcome to the [Minehut](https://minehut.com) Meta GitHub repo! This is *the* place to **report bugs** and **suggest features and improvements** for Minehut, the server hosting network! You can also participate in conversations about feature requests and bug reports. 
 
-We would like to improve communication with our users by providing official channels to report problems and tell us your ideas. This space will be taken as an (experimental) approach to see if you feel comfortable reporting and tracking your issues by means of this tool. This document (called the README) will serve as a starting point for making your first issue.
+We would like to improve communication with our users by providing official channels to report problems and tell us your ideas. This space will be taken as an (experimental) approach to see if you feel comfortable reporting and tracking your issues utilizing this tool. This document (called the README) will serve as a starting point for making your first issue.
+
+A lot of conversations and announcements happen on our discord, which you can join [here](https://discord.gg/pgXMSFh). Many longer conversations about issues happen there, as well as diving into other topics such as changing the staff training process. 
 
 ## Contents
 - [General Introduction](https://github.com/Minehut/Meta#general-introduction)
 - [Community Expectations](https://github.com/Minehut/Meta#community-expectations)
 - [Submitting An Issue](https://github.com/Minehut/Meta#submitting-an-issue)
 
 ## General Introduction
-This is the Minehut Meta GitHub repository. Here, you can tell us about your features and ideas that you want to be added to Minehut and you can also report bugs that need to be squashed. We use GitHub's Issue Tracker feature to track bug reports and feature requests (suggestions).
+This is the Minehut Meta GitHub repository. Here, you can tell us about your features and ideas that you want to be added to Minehut and report bugs that need to be squashed. We use GitHub's Issue Tracker feature to track bug reports and feature requests (suggestions).
 
 We understand you might not have much experience using GitHub, and that's fine! We'll do our best to help you navigate this tracker. We can start by introducing some key terms you'll be hearing a lot:
 - **Issue**: A feature request or **bug** report.
 - **Repository** or **repo**: Refers to the group of Minehut Meta pages. Includes the **issue** tracker and documents (like this one).
-- **Bug**: Any piece of unintended behaviour anywhere on Minehut. Basically, if something isn't working how it should, it's a bug.
+- **Bug**: Any piece of unintended behavior anywhere on Minehut. Basically, if something isn't working how it should, it's a bug.
```

Comment:
What's up with this one? Behaviour is correct so I'm not sure why you changed it.